docs/user/user_doc.md

```diff
@@ -53,6 +53,111 @@ poll_url = start_response["url"]
 poll_response = model.poll(poll_url)
 ```
 
+### Uploading Models
+In addition to exploring and running models, the aiXplain SDK allows you to upload your own models to the aiXplain platform. This requires a working model image in line with the template specified [here](https://github.com/aixplain/aiXplain/blob/M-5079254326-container-upload-cli/docs/user/model_setup.md). You will also be required to have an aiXplain account as well as a TEAM_API_KEY which should be set either as an environment variable or passed into each of the following functions.
+
+First, choose a hosting machine appropriate for your model. Note down the host machines "code". You can list the available hosting machines' specifications by running the following:
+```console
+$ aixplain list hosts [--api-key <TEAM_API_KEY>]
+[
+    {
+        "id": "64dce914adc92335dc35beb5",
+        "code": "aix-2c-8g-od",
+        "type": "on-demand",
+        "cores": 2,
+        "memory": 8,
+        "hourlyCost": 0.12
+    },
+    {
+        "id": "64dceafdadc92335dc35beb6",
+        "code": "aix-2c-8g",
+        "type": "always-on",
+        "cores": 2,
+        "memory": 8,
+        "hourlyCost": 0.096
+    },
+    ...
+]
+```
+
+Find a supported function type that best describes your model's purpose. Note down the function's ID.
+```console
+$ aixplain list functions [--api-key <TEAM_API_KEY>]
+{
+    "total": 55,
+    "pageTotal": 55,
+    "items": [
+        {
+            "id": "language-identification",
+            "name": "Language Identification",
+            "output": [
+                {
+                    "name": "Label",
+                    "code": "data",
+                    "dataType": "label",
+                    "defaultValue": []
+                }
+            ],
+            "params": [
+                {
+                    "code": "text",
+                    "name": "Text",
+                    "required": true,
+                    "isFixed": false,
+                    "dataType": "text",
+                    "dataSubType": "text",
+                    "multipleValues": false
+                }
+            ]
+        },
+        ...
+    ]
+}
+```
+
+Once you have chosen a suitable host machine and function, register your model and create an image repository:
+
+```console
+$ aixplain create repo --name <model_name> --hosting-machine <machine_code> --version <model_version> --description <model_description> --function <function_id> --is-async <model_is_asynchronous or not> --source-language <source_language> [--api-key <TEAM_API_KEY>]
```

Comment:
- name: We don't have that yet. Let me check.
- function: I'm not sure; I'll check with the backend
- is-async: Yes, this takes boolean arguments. I'll hard-code it to False for this first release.
- source-language: I'm not sure. I'll check with Hadi. I think we had that field just because we were trying to get ASR to work.